Explorer++/Helper/FileOperations.cpp

```diff
@@ -90,17 +102,16 @@ HRESULT FileOperations::DeleteFiles(HWND hwnd, const std::vector<PCIDLIST_ABSOLU
 		}
 	}
 
-	wil::com_ptr_nothrow<IShellItemArray> shellItemArray;
-	hr = SHCreateShellItemArrayFromIDLists(static_cast<UINT>(pidls.size()), &pidls[0],
-		&shellItemArray);
+	wil::com_ptr_nothrow<IShellItemArray> shellItemArray{};
+	hr = SHCreateShellItemArrayFromIDLists(static_cast<UINT>(pidls.size()), &pidls[0], &shellItemArray);
 
 	if (FAILED(hr))
 	{
 		return hr;
 	}
 
 	wil::com_ptr_nothrow<IUnknown> unknown;
-	hr = shellItemArray->QueryInterface(IID_IUnknown, reinterpret_cast<void **>(&unknown));
+	hr = shellItemArray->QueryInterface(IID_IUnknown, reinterpret_cast<void**>(&unknown));

```

Comment:
I know you didn't add this, but it would be useful to update this line to use `IID_PPV_ARGS`. That is, this should be changed to:

```c++
hr = shellItemArray->QueryInterface(IID_PPV_ARGS(&unknown));
```